Makefile

```diff
@@ -24,6 +24,9 @@ export WATCH_NAMESPACE?=openshift-sriov-network-operator
 export GOFLAGS+=-mod=vendor
 export GO111MODULE=on
 PKGS=$(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
+# github.com/jetstack/cert-manager latest version
+export CERT_MANAGER_VERSION?=v1.7.0
+export CERT_MANAGER?=true
```

Comment:
nit: `ENABLE_CERT_MANAGER` ?  like we have `ENABLE_ADMISSION_CONTROLLER`

Reply:
makes a lot of sense. Thanks